akka-stream/src/main/scala/akka/stream/OperationAttributes.scala

```diff
@@ -1,20 +1,30 @@
 /**
  * Copyright (C) 2014 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.stream.scaladsl
+package akka.stream
 
+import akka.stream.javadsl.japi
 import akka.stream.impl.Stages.StageModule
-import akka.stream.Supervision
 
 /**
  * Holds attributes which can be used to alter [[Flow]] or [[FlowGraph]]
  * materialization.
+ *
+ * Note that more attributes for the [[ActorFlowMaterializer]] are defined in [[ActorOperationAttributes]].
  */
 final case class OperationAttributes private (attributes: List[OperationAttributes.Attribute] = Nil) {
 
   import OperationAttributes._
 
   /**
+   * Java API
+   */
+  def getAttributes(): java.util.List[Attribute] = {
```

Comment:
actually scratch the second one: we should be more opinionated and offer these two
~~~scala
def getAttributes[T <: Attribute](c: Class[T]): java.util.List[T]
def getAttribute[T <: Attribute](c: Class[T], default: T): T
~~~